python/modules/IcePy/Util.cpp

```diff
@@ -231,30 +231,26 @@ IcePy::PyObjectHandle::PyObjectHandle(PyObject* p) : _p(p) {}
 
 IcePy::PyObjectHandle::PyObjectHandle(const PyObjectHandle& p) : _p(p._p) { Py_XINCREF(_p); }
 
-IcePy::PyObjectHandle::~PyObjectHandle()
-{
-    if (_p)
-    {
-        Py_DECREF(_p);
-    }
-}
+IcePy::PyObjectHandle::~PyObjectHandle() { Py_XDECREF(_p); }
```

Comment:
`Py_XDECREF` and `Py_XINCREF` include null handling.

See https://docs.python.org/3/c-api/refcounting.html